controller/manager/manager.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manager
+
+import (
+	"fmt"
+
+	operatorv1alpha1 "github.com/projectcontour/contour-operator/api/v1alpha1"
+	contourcontroller "github.com/projectcontour/contour-operator/controller/contour"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
It was included in the [kubebuilder bootstrap](https://book.kubebuilder.io/cronjob-tutorial/empty-main.html). I simply moved it to this manager pkg.

Reply:
I removed `_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"` since the linter complained:
```
controller/manager/manager.go:26:2: a blank import should be only in a main or test package, or have a comment justifying it (golint)
	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```